### PR TITLE
Update branding to 6.3.3

### PR DESCRIPTION
**Pull request by JunTaoLuo**
https://github.com/aspnet/AspNetCore-Internal/issues/3464

### CODE REVIEW

**Comment by JunTaoLuo** [comment]
This is ready to go

**Comment by JunTaoLuo** [comment]
@dougbu  @wtgodbe please review

File: azure-pipelines.yml [hunk 1]
```diff
@@ -9,6 +9,14 @@ variables:
     value: true
   - name: _DotNetArtifactsCategory
     value: ASPNETENTITYFRAMEWORK6
+  - ${{ if ne(variables['System.TeamProject'], 'public') }}:
+    - group: DotNet-MSRC-Storage
```
Comment (wtgodbe) [inline azure-pipelines.yml]: does this need to be reverted @mmitche?

Comment (mmitche) [reply azure-pipelines.yml]: Nope

File: eng/Versions.props [hunk 2]
```diff
@@ -31,7 +31,7 @@
   <PropertyGroup Label="Dependencies from dotnet/core-setup">
     <MicrosoftNETCoreAppRefPackageVersion>3.0.0</MicrosoftNETCoreAppRefPackageVersion>
     <MicrosoftNETCoreAppRuntimewinx64PackageVersion>3.0.2</MicrosoftNETCoreAppRuntimewinx64PackageVersion>
-    <MicrosoftNETCoreAppInternalPackageVersion>3.0.2-servicing-19604-02</MicrosoftNETCoreAppInternalPackageVersion>
+    <MicrosoftNETCoreAppInternalPackageVersion>3.0.2</MicrosoftNETCoreAppInternalPackageVersion>
```
Comment (mmitche) [inline eng/Versions.props]: This shouldn't be changed.  I'm investigating why it needed to change.

**Comment by mmitche** [comment]
I put the proper non-shipping blobs into the right container so that you can revert the NETCoreApp.Internal version change.

**Comment by JunTaoLuo** [comment]
https://dev.azure.com/dnceng/public/_build/results?buildId=482835&view=logs&j=0bc77094-9fcd-5c38-f6e4-27d2ae131589&t=108c10b7-2c4b-5297-f2c3-f882c6024b18

I see failures so I thought I needed to update it.

**Comment by JunTaoLuo** [comment]
For example https://dotnetcli.azureedge.net/dotnet/Runtime/3.1.1-servicing.19608.4/dotnet-win-x64.3.1.1-servicing.19608.4.zip
gives a 404

File: eng/Versions.props (hunk 2 above)
Comment (JunTaoLuo) [reply eng/Versions.props]: Uh oh, I may need to revert this change on every other PR I made. We'll need to figure out why this is 404ing the build first.

**Comment by mmitche** [comment]
> For example https://dotnetcli.azureedge.net/dotnet/Runtime/3.1.1-servicing.19608.4/dotnet-win-x64.3.1.1-servicing.19608.4.zip
> gives a 404
> image

I had forgotten to move the non-shipping binaries over.

https://dotnetcli.azureedge.net/dotnet/Runtime/3.1.1-servicing.19608.4/dotnet-runtime-3.1.1-servicing.19608.4-win-x64.zip

**Comment by mmitche** [comment]
It should not 404 from here on out.

**Comment by JunTaoLuo** [comment]
update @dougbu

**Pull request merged**